### ISSUE TITLE
RQ 1.5.0 client not working with fakeredis

### ISSUE THREAD

**Issue by omry** [jamesls/fakeredis#273]
Probably related to this item from the [release notes](https://github.com/rq/rq/releases/tag/v1.5.0):

> RQ is now aware of which version of Redis server it's running on

```
self = FakeConnection<server=<fakeredis._server.FakeServer object at 0x7f0374be3c70>,db=0>                                                                                                                                                                                                                                                     
                                                                                                                                                                                                                                                                                                                                               
    def read_response(self):                                                                                                                                                                                                                                                                                                                   
        if not self._server.connected:                                                                                                                                                                                                                                                                                                         
            try:                                                                                                                                                                                                                                                                                                                               
                response = self._sock.responses.get_nowait()                                                                                                                                                                                                                                                                                   
            except queue.Empty:                                                                                                                                                                                                                                                                                                                
                raise redis.ConnectionError(CONNECTION_ERROR_MSG)                                                                                                                                                                                                                                                                              
        else:                                                                                                                                                                                                                                                                                                                                  
            response = self._sock.responses.get()                                                                                                                                                                                                                                                                                              
        if isinstance(response, redis.ResponseError):                                                                                                                                                                                                                                                                                          
>           raise response                                                                                                                                                                                                                                                                                                                     
E           redis.exceptions.ResponseError: unknown command 'INFO'    
```

**Comment by bmerry** [comment]
That's expected: INFO is one of the commands documented as not implemented by fakeredis. You're welcome to submit a pull request to implement it.

It's not clear what version is appropriate to return, because fakeredis is missing a hodgepodge of commands, so doesn't exactly match any redis version. A version string that will make RQ work right might not be correct for other packages. One might be better off mocking `get_redis_server_version` in RQ.

**Comment by omry** [comment]
This seems like it's making fakeredis unusable so I would assume it's a high priority issue.

As for what version to return:
I think the most compatible version at minimum (where there is the most coverage).
an API to change the returned version might also be useful, but I am not sure it's worth it.

**Comment by untitaker** [comment]
Just came across this issue. In my opinion RQ should handle this case explicitly as it can make a better decision about what to do in case of there not *being* a server.

**Comment by jan-matthis** [comment]
This got addressed in RQ (included in v1.5.1): https://github.com/rq/rq/commit/6028a636073909c6472703f4c42b610a70b1f889, the issue can probably be closed

**Comment by bmerry** [comment]
Thanks for letting me know. Closing.
